### ISSUE TITLE
AgentHotelplusid

### ISSUE THREAD

**Issue by Penjaraisreal** [Penjaraisreal/Aaaaaa#3]
INGIN BERTEMAN DENGAN PARA WANITA DIBERBAGAI TEMPAT.
CEKIN HOTEL MUDAH LENGKAP DENGAN SERVIS LEDIES DIDALAMNYA 
<title>https://www.instagram.com/sherlyirenee?igsh=OGQ5ZDc2ODk2ZA==
<link href="style.css" rel="stylesheet" type="text/css" />UNTUK DAFTAR PILIHAN TEMPAT YANG NYAMAN DAN HARGA TERJANGKAU 
<link href="js/js-image-slider.css" rel="stylesheet" type="text/css" />
<script src="js/js-image-slider.js" type="text/javascript"></script>
<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Bitter:400,700">
<link href="generic.css" rel="stylesheet" type="text/css" />
<link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/themes/base/jquery-ui.css" rel="stylesheet" type="text/css"/>
  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>
  <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/jquery-ui.min.js"></script>
  <link rel="stylesheet" media="screen" href="calendar.css"  />
  <script>
  $(document).ready(function() {
    $("#datepicker").datepicker();
  });
    $(document).ready(function() {
    $("#datepicker2").datepicker();
  });
  </script>
  <link href="select2.css" rel="stylesheet"/>
    <script src="jquery-1.8.0.min.js"></script>
    <script src="select2.js"></script>
    <script>
        $(document).ready(function() {
            $("#states").select2();   
        });
    </script>
</head>
<body>
<div class="konten">
<div class="Menu">
<ul id="pilihan">
<li id="log"><img src="ing/lokec.PNG" style="width:28px"></li>
<a href=""><li id="home">HOME</li></a>
<a href=""><li id="shop">SHOP</li></a>
<a href=""><li id="land">MINGGO LAND</li></a>
<a href=""><li id="about">ABOUT US</li></a>
</ul>
</div>

<div class="tengah">

<div class="banner_back">
<div class="slider2">
 <div id="sliderFrame">
        <div id="slider">
            <a href="#" target="_blank">
                <img src="ing/koki.jpg" style="width:100%" alt="Selamat Datang di Minggo.co.id" />
            </a>
            <img src="ing/kok.png" style="width:100%" alt="" />
            <img src="ing/ko.png" style="width:100%" alt="" />
            <img src="ing/koki.jpg" style="width:100%" alt="" />
            <img src="ing/kok.png" style="width:100%" />
        </div>
    </div>
</div>

<div class="tiket">
<div id="tab">
<h3>Hotel</h3>
</div>
<div id="judul">
<h1>Cari Hotel Termurah Jawa Barat Disini</h1>
</div>
<table id="tiket_tab">
 <tr>
 <form action="proses.php" name="form_tiket">
  <td id="carip">
   <div>
   <h2>1. Pilih Tujuan Wisata</h2>
   <p>Tempat Wisata, daerah, nama hotel</p>
   <select style="width:100%" id="states">
               <optgroup label="Pantai">
                   <option value="AK">Pangandaran</option>
                   <option value="HI">Santolo</option>
       <option value="AK">Sayang Heulang</option>
                   <option value="HI">Manalusu</option>
       <option value="AK">Rancabuaya</option>
               </optgroup>
               <optgroup label="Pegunungan">
                   <option value="CA">Cikurai</option>
                   <option value="NV">Papandayan</option>
                   <option value="OR">Ciremai</option>
                   <option value="WA">Guntur</option>
               </optgroup>
               <optgroup label="Danau">
                   <option value="AZ">Situ Bagendit</option>
                   <option value="CO">Cangkuang</option>
                   <option value="ID">Talaga Bodas</option>
                   <option value="MT">Kawah Putih</option><option value="NE">Nebraska</option>
               </optgroup>
              </select>
   </div>
  </td>
  <td style="width:30%">
   <div>
   <h2>2. Tentukan Waktu Menginap</h2>
   <p>Check In</p>
   <input type="text" id="datepicker" name="waktu_datang" placeholder="01/06/2015">
   <p>Berapa Lama?</p>
   <input type="text" name="waktu" placeholder="2 Hari">
   <p>Check Out</p>
   <input type="text" id="datepicker2" name="waktu_akhir" placeholder="01/08/2015">
   </div>
  </td>
  <td style="width:10%">
   <div>
   <h2>3. Mulai Pencarian</h2>
   <input type="submit" name="submit" value="Cari">
   </div>
  </td>
 </form>
 </tr>
</table>
</div>
</div>

<div class="profil">
<h1>Mengapa pilih Kami? Ini alasannya </h1>
<table id="tab_profil">
<tr id="tab_ul">
<td>
<img src="ing/logos.png" style="width: 80px;">
</td>
<td>
<h3>Judul 1</h3>
<span>dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad</span>
</td>
</tr>

<tr id="tab_ul">
<td>
<img src="ing/logos.png" style="width: 80px;">
</td>
<td>
<td>
<h3>Judul 2</h3>
<span>dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad</span>
</td>
</tr>

<tr id="tab_ul">
<td>
<img src="ing/logos.png" style="width: 80px;">
</td>
<td>
<h3>Judul 3</h3>
<span>dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad</span>
</td>
</tr>

</table>
</div>

<div class="profil2">
<table id="tab_profil2" style="width:100%;height: 65px;">

<tr id="tab_ul2" style="width: 65%;padding-left: 10px;padding-top: 15px;">
<td>
<h2>Dapatkan informasi terbaru harga tiket promo melalui email Anda.</h2>
</td>
</tr>

<tr id="tab_ul2">
<td>
<input type="text" name="email" placeholder="Masukkan Email" style="background: white;width: 240px;height: 40px;border: 1px solid #FFF;color: #545454;"/>
<input type="submit" name="submit" value="Kirim" style="background:rgb(102, 102, 102)"/>

</td>
</tr>
</table>
</div>

<div class="maindiv">
<div class="sidebar1">
<h2>on Facebook</h2>
<iframe src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2Fgapuraofficial&amp;width=320&amp;height=258&amp;colorscheme=light&amp;show_faces=true&amp;header=false&amp;stream=false&amp;show_border=true&amp;appId=1500968830142627" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:320px; height:258px;" allowTransparency="true"></iframe>
<br/>
<br/>
<h2>on Twitter</h2>
<br/>
<a href="https://twitter.com/gapuratv" class="twitter-follow-button" data-show-count="false" data-size="large" data-dnt="true">Follow @gapuratv</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
</div>

<div class="kalender" style="float: right;width: 60%;">
<form method="post">
<table border="0" cellpadding="2" cellspacing="2">
  <tr>
    <td colspan="3" align="left" valign="middle"><?php
//Draw Calendar
function draw_calendar($month,$year){

 // Draw table for Calendar 
 $calendar = '<table cellpadding="0" cellspacing="0" class="calendar">';

 // Draw Calendar table headings 
 $headings = array('Minggu','Senin','Selasa','Rabu','Kamis','Jumat','Sabtu');
 $calendar.= '<tr class="calendar-row"><td class="calendar-day-head">'.implode('</td><td class="calendar-day-head">',$headings).'</td></tr>';

 //days and weeks variable for now ... 
 $running_day = date('w',mktime(0,0,0,$month,1,$year));
 $days_in_month = date('t',mktime(0,0,0,$month,1,$year));
 $days_in_this_week = 1;
 $day_counter = 0;
 $dates_array = array();

 // row for week one 
 $calendar.= '<tr class="calendar-row">';

 // Display "blank" days until the first of the current week 
 for($x = 0; $x < $running_day; $x++):
  $calendar.= '<td class="calendar-day-np">&nbsp;</td>';
  $days_in_this_week++;
 endfor;

 // Show days.... 
 for($list_day = 1; $list_day <= $days_in_month; $list_day++):
  if($list_day==date('d') && $month==date('n'))
  {
   $currentday='currentday';
  }else
  {
   $currentday='';
  }
  $calendar.= '<td class="calendar-day '.$currentday.'">';
  
   // Add in the day number
   if($list_day<date('d') && $month==date('n'))
   {
    $showtoday='<strong class="overday">'.$list_day.'</strong>';
   }else
   {
    $showtoday=$list_day;
   }
   $calendar.= '<div class="day-number">'.$showtoday.'</div>';

  // Draw table end
  $calendar.= '</td>';
  if($running_day == 6):
   $calendar.= '</tr>';
   if(($day_counter+1) != $days_in_month):
    $calendar.= '<tr class="calendar-row">';
   endif;
   $running_day = -1;
   $days_in_this_week = 0;
  endif;
  $days_in_this_week++; $running_day++; $day_counter++;
 endfor;

 // Finish the rest of the days in the week
 if($days_in_this_week < 8):
  for($x = 1; $x <= (8 - $days_in_this_week); $x++):
   $calendar.= '<td class="calendar-day-np">&nbsp;</td>';
  endfor;
 endif;

 // Draw table final row
 $calendar.= '</tr>';

 // Draw table end the table 
 $calendar.= '</table>';
 
 // Finally all done, return result 
 return $calendar;
}

echo '<h2>'.date('M').' '.date('Y').'</h2>';
echo draw_calendar(date('n'),date('Y'));
?></td>
  </tr>
  </table>
</form>
</div>
</div>

</div>
<div class="create_web">
<div id="box">
<a href="#"><img src="ing/create.png" style="width:100%"/></a>
</div>
</div>
</div>

</div>
</body>
<footer>
<div class="bawah">
<div class="k1">
<h5>Our Contact</h5>
<h4 id="foot_text" style="font-weight:bold">+62895321100404</h4>
<h4 id="foot_text" style="font-weight:bold">+62895321100404</h4>
<h4 id="foot_text">Intan Regency, Blok K No.7 Garut</h4>
</div>
https://www.instagram.com/sherlyirenee?igsh=OGQ5ZDc2ODk2ZA==
<div class="k2">
</div>

<div class="k3">
<img id="logo_bawah1" src="ing/logo_bawah.png"/>
<img id="logo_bawah2" src="ing/logo_bawah2.png"/>
<img id="logo_bawah2" src="ing/logo_bawah3.png"/>
<img id="logo_bawah1" src="ing/logo_bawah1.png"/>
</div>
</div>
</footer>
</html>

**Comment by Penjaraisreal** [comment]
> INGIN BERTEMAN DENGAN PARA WANITA DIBERBAGAI TEMPAT. CEKIN HOTEL MUDAH LENGKAP DENGAN SERVIS LEDIES DIDALAMNYA
> 
> <title>https://www.instagram.com/sherlyirenee?igsh=OGQ5ZDc2ODk2ZA== UNTUK DAFTAR PILIHAN TEMPAT YANG NYAMAN DAN HARGA TERJANGKAU <script src="js/js-image-slider.js" type="text/javascript"></script> <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script> <script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/jquery-ui.min.js"></script> <script> $(document).ready(function() { $("#datepicker").datepicker(); }); $(document).ready(function() { $("#datepicker2").datepicker(); }); </script> <script src="jquery-1.8.0.min.js"></script> <script src="select2.js"></script> <script> $(document).ready(function() { $("#states").select2(); }); </script>
> * ![](ing/lokec.PNG)
> * HOME * SHOP * MINGGO LAND * ABOUT US
> 
> [ ![Selamat Datang di Minggo.co.id](ing/koki.jpg) ](#) ![](ing/kok.png) ![](ing/ko.png) ![](ing/koki.jpg) ![](ing/kok.png)
> ### Hotel
> # Cari Hotel Termurah Jawa Barat Disini
> ## 1. Pilih Tujuan Wisata
> Tempat Wisata, daerah, nama hotel
> 
> Pangandaran Santolo Sayang Heulang Manalusu Rancabuaya Cikurai Papandayan Ciremai Guntur Situ Bagendit Cangkuang Talaga Bodas Kawah PutihNebraska
> ## 2. Tentukan Waktu Menginap
> Check In
> 
> Berapa Lama?
> 
> Check Out
> 
> ## 3. Mulai Pencarian
> # Mengapa pilih Kami? Ini alasannya
> ![](ing/logos.png)	
> ### Judul 1
> dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad
> ![](ing/logos.png)		
> ### Judul 2
> dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad
> ![](ing/logos.png)	
> ### Judul 3
> dwkahdbkw jndkjan wkjdkjaw nkdjnakjw ndkad jdkwkad
> ## Dapatkan informasi terbaru harga tiket promo melalui email Anda.
> ## on Facebook
> <iframe src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2Fgapuraofficial&width=320&height=258&colorscheme=light&show_faces=true&header=false&stream=false&show_border=true&appId=1500968830142627" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:320px; height:258px;" allowTransparency="true"></iframe>
> 
> ## on Twitter
> 
> [Follow @gapuratv](https://twitter.com/gapuratv) <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
> // Draw table for Calendar
> $calendar = '';
> // Draw Calendar table headings $headings = array('Minggu','Senin','Selasa','Rabu','Kamis','Jumat','Sabtu'); $calendar.= '
> 
> ';
> //days and weeks variable for now ... $running_day = date('w',mktime(0,0,0,$month,1,$year)); $days_in_month = date('t',mktime(0,0,0,$month,1,$year)); $days_in_this_week = 1; $day_counter = 0; $dates_array = array();
> 
> // row for week one $calendar.= '
> 
> ';
> // Display "blank" days until the first of the current week for($x = 0; $x < $running_day; $x++): $calendar.= '
> 
> ';
> $days_in_this_week++;
> endfor;
> // Show days.... for($list_day = 1; $list_day <= $days_in_month; $list_day++): if($list_day==date('d') && $month==date('n')) { $currentday='currentday'; }else { $currentday=''; } $calendar.= '
> 
> ';
> if($running_day == 6):
> $calendar.= '';
> if(($day_counter+1) != $days_in_month):
> $calendar.= '';
> endif;
> $running_day = -1;
> $days_in_this_week = 0;
> endif;
> $days_in_this_week++; $running_day++; $day_counter++;
> endfor;
> // Finish the rest of the days in the week if($days_in_this_week < 8): for($x = 1; $x <= (8 - $days_in_this_week); $x++): $calendar.= '
> 
> ';
> endfor;
> endif;
> // Draw table final row $calendar.= '
> 
> ';
> // Draw table end the table $calendar.= '
> 
> '.implode('	',$headings).'
>  	';
> // Add in the day number if($list_day<date('d') && $month==date('n')) { $showtoday='**'.$list_day.'**'; }else { $showtoday=$list_day; } $calendar.= '
> 
> '.$showtoday.'
> ';
> // Draw table end $calendar.= '
> 
>  
> ';
> // Finally all done, return result return $calendar; }
> 
> echo '
> 
> ## '.date('M').' '.date('Y').'
> ';
> echo draw_calendar(date('n'),date('Y'));
> ?>
> [![](ing/create.png)](#)
> ##### Our Contact
> #### +62895321100404
> #### +62895321100404
> #### Intan Regency, Blok K No.7 Garut
> https://www.instagram.com/sherlyirenee?igsh=OGQ5ZDc2ODk2ZA==
> ![](ing/logo_bawah.png) ![](ing/logo_bawah2.png) ![](ing/logo_bawah3.png) ![](ing/logo_bawah1.png)
